vaadin-grid-pro-flow-parent/vaadin-grid-pro-flow/src/main/resources/META-INF/resources/frontend/gridProConnector.js

```diff
@@ -17,6 +17,13 @@
   }
 
   window.Vaadin.Flow.gridProConnector = {
+    focusCustomEditor: (editor) => {
```

Comment:
I wasn't sure about the name, so I got "inspired" by the name of the [method this behavior was taken from](https://github.com/vaadin/web-components/blob/2a51598e6db4bd4fa441fbe3b26030f82f2c3242/packages/grid-pro/src/vaadin-grid-pro-edit-column.js#L198-L207):

```js
  _focusEditor(editor) {
    editor.focus();
    if (this.editorType === 'checkbox') {
      editor.setAttribute('focus-ring', '');
    } else if (editor instanceof HTMLInputElement) {
      editor.select();
    } else if (editor.focusElement && editor.focusElement instanceof HTMLInputElement) {
      editor.focusElement.select();
    }
  }
```

Changed to `selectAll(editor)` since it makes more sense.